Sources/AST/Expression/ExternalCall.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ExternalCall.swift
+//  AST
+//
+//  Created by Catalin Craciun on 10/11/2018.
+//
+import Source
+import Lexer
+
+/// A call to an external function
+public struct ExternalCall: ASTNode {
+  public enum Mode {
+    case normal, returnsGracefullyOptional, isForced
```

Comment:
What's your opinion on this vs @Aurel300's approach with the Token?

Reply:
I don't think it's necessary to keep the token, since it's so basic.

In general, we should put as much functionality on the AST nodes as possible. We can have the token and maybe a computed property for mode.